linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_passwordauth/tests/authselect_correct_value.pass.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# packages = authselect
+# platform = Red Hat Enterprise Linux 8,Red Hat Enterprise Linux 9,multi_platform_fedora
+
+authselect create-profile hardening -b sssd
+CUSTOM_PROFILE="custom/hardening"
+authselect select $CUSTOM_PROFILE --force
+CUSTOM_SYSTEM_AUTH="/etc/authselect/$CUSTOM_PROFILE/system-auth"
```

Comment:
copy-pasta error, this should set `password-auth`

Reply:
Actually, I inverted this script between `set_password_hashing_algorithm_passwordauth` and set_password_hashing_algorithm_systemauth`. Thanks for noticing this. I will adjust both.